### PR TITLE
add: `megabasterd-bin`

### PR DESCRIPTION
**Pull request by vigress8**

### CODE REVIEW

**Review by Elsie19 — CHANGES_REQUESTED** [review]
Add to `packagelist`.

File: packages/megabasterd-bin/megabasterd-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name="megabasterd-bin"
+gives="megabasterd"
+_name="MegaBasterd"
```
Comment (Elsie19) [inline packages/megabasterd-bin/megabasterd-bin.pacscript:3]: Remove that and just use the actual string instead.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/megabasterd-bin/megabasterd-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name="megabasterd-bin"
+gives="megabasterd"
+pkgver="8.21"
+pkgdesc="Yet another unofficial (and ugly) cross-platform MEGA downloader/uploader/streaming suite"
+homepage="https://github.com/tonikelope/${gives}"
+depends=("java11-runtime")
```
Comment (Elsie19) [inline packages/megabasterd-bin/megabasterd-bin.pacscript:6]: Does it only work on java11?

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/megabasterd-bin/megabasterd-bin.pacscript (hunk 2 above)
Comment (vigress8) [reply packages/megabasterd-bin/megabasterd-bin.pacscript:6]: Java11 or later. But some of the GitHub runners were pulling Java8 to satisfy the `java-runtime` dependency

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**